layers/ubuntu1604/bazel/BUILD

```diff
@@ -47,6 +50,30 @@ download_pkgs(
     ],
 )
 
+# Transform the packages metadata csv produced by download_pkgs into a YAML
+# file.
+packages_metadata(
+    name = "bazel_debs_yaml_metadata",
+    metadata_csv = ":bazel_debs_metadata.csv",
+)
+
+# Run a vulnerability check on the bazel image and generate a YAML file with
+# a list of vulnerability tags.
+security_check(
+    name = "bazel_debs_security_check",
+    image = "gcr.io/asci-toolchain/nosla-ubuntu16-04-bazel",
```

Comment:
why not the public one?

Reply:
Because we don't have the permission to run advanced analysis on that image.